test/integration/commands/excel2json/test_json_header.py

```diff
@@ -0,0 +1,62 @@
+import json
+from pathlib import Path
+
+import pytest
+import regex
+
+from dsp_tools.commands.excel2json.json_header import get_json_header
+from dsp_tools.commands.excel2json.models.json_header import EmptyJsonHeader
+from dsp_tools.commands.excel2json.models.json_header import FilledJsonHeader
+from dsp_tools.commands.excel2json.models.json_header import JsonHeader
+from dsp_tools.models.exceptions import InputError
+
+
+def test_get_json_header_no_file() -> None:
+    test_path = Path("testdata/excel2json/excel2json_files")
+    result = get_json_header(test_path)
+    assert isinstance(result, EmptyJsonHeader)
+
+
+@pytest.fixture()
+def filled_json_header() -> JsonHeader:
+    test_path = Path("testdata/excel2json/new_excel2json_files/json_header.xlsx")
+    return get_json_header(test_path)
```

Comment:
just as a matter of taste: I would put all fixtures above the first test case. IMO it makes navigating the file easier.